docs explaining about current, crashed, handled and level fields (#1581)
getsentry/sentry-docs: src/collections/_documentation/development/sdk-dev/event-payloads/threads.md

`crashed`:

> : _Optional_. A flag indicating whether the thread has caused the error, it doesn't mean that the Application has been terminated, but captured the error. Defaults to `false`.

Reviewer: ```suggestion
: _Optional_. A flag indicating whether this thread has caused the error. It does not necessarily indicate the application terminated. Defaults to `false`.
```

> It may be seen as `crashed: yes` in the Issues page.

Reviewer 2: ```suggestion
: _Optional_. A flag indicating whether this thread is the one which captured the event sent to Sentry. In spite of the name, it doesn't necessarily indicate that the application terminated. Defaults to `false`, and should only be set to `true` on one thread.
On the Issue Details page, appears as a `crashed: yes` tag on the thread in question, and causes that thread to be the one shown by default.
```

`crashed`:

> : _Optional_. A flag indicating whether this thread has caused the error. It does not necessarily indicate the application terminated. Defaults to `false`.

Reviewer 3: ```suggestion
: _Optional_. A flag indicating whether this thread has caused the error. It does not necessarily indicate the application terminated -- defaults to `false`.
```

> It may be seen as `crashed: yes` in the Issues page.

Reviewer 3: ```suggestion
It may display as `crashed: yes` on the Issues page.
```

Reviewer 4: Both options here ("may be seen" and "may display") suggest that it may or may not display like that. Is there a case where the event has `crashed: true` and doesn't display `crashed: yes` in Sentry?!

Why do we say "may"?

Reviewer 4: If the usage of "may" was future-proofing in case of UI changes, then maybe we should not mention it altogether?

Reviewer 3: Great question! I was assuming that we included the word "may" because it was uncertain? Happy to take your lead on this and omit the word "may." @marandaneto do you know if there's a reason to emphasize "may?"

Author: @MimiDumpling tbh no, it's just a synonym of `can` that didn't work well, so the meaning would be: `can be seen/it can be viewed`, I've changed to `it displays as...` :)

Author: @bruno-garcia true, we can talk about this on TSC again, I generally agree, but still think the current status of docs would need to be changed, either by this PR or other, this is a very recurrent problem.
I'll keep this PR open then.

> It may display as `crashed: yes` on the Issues page.

Author: ```suggestion
It displays as `crashed: yes` on the Issues page.
```

> When this flag is set to `true` it's also used to preselect a thread in the thread's selector on the Issues page.

Reviewer 3: ```suggestion
When you set this flag to `true`, it's also used to preselect a thread in the thread's selector on the Issues page.
```

`current`:

> : _Optional_. A flag indicating whether the thread was in the foreground, often known as UI or main thread. Defaults to `false`.

Reviewer 3: ```suggestion
: _Optional_. A flag indicating whether the thread was in the foreground, often known as UI or main thread -- defaults to `false`.
```

> It may be seen as `was active: yes` in the Issues page.

Reviewer 3: ```suggestion
It may display as `was active: yes` on the Issues page.
```

> It may display as `was active: yes` on the Issues page.

Author: ```suggestion
It displays as `was active: yes` on the Issues page.
```
